internal/server/handlers/validation/database_cluster.go

```diff
@@ -544,6 +544,11 @@ func (h *validateHandler) validateDatabaseClusterOnUpdate(
 		return errCannotChangeStorageSize
 	}
 
+	// Do not allow shrinking storage size.
```

Comment:
the condition on line 543 currently prevents this code from being executed, but it is going to be removed eventually once support for storage expansion is merged. The above condition currently exists only temporarily to isolate the various FBs